### PR TITLE
updated What is the best time to CoinJoin section

### PR DESCRIPTION
**Pull request by peterchaudhary**
To learn how to make changes, changed CoinJoin to coinjoin.

### CODE REVIEW

**Review by MarnixCroes — APPROVED** [review]
well done! 
congrats on your first Pull Request

File: docs/FAQ/FAQ-UseWasabi.md [hunk 1]
```diff
@@ -766,13 +766,13 @@ For example, with a 0.7 BTC input you would expect the following outputs: ~0.1,
...
-That said, the ideal way to CoinJoin would be when:
+You can coinjoin whenever you want.
+That said, the ideal way to coinjoin would be when:
 
 1. Mempool is clean.
-CoinJoins are unfairly cheap when the mempool is empty.
+Coinjoins are unfairly cheap when the mempool is empty.
 Remember to take a look at websites like [Johoe's Bitcoin Mempool Statistics](https://jochen-hoenicke.de/queue/#0,24h) or [mempool.space](https://mempool.space/tv) to see mempool statistics in real time.
 
 2. Your inputs are the same or little higher as the minimal denomination for the current round.
```
Comment (MaxHillebrand) [inline docs/FAQ/FAQ-UseWasabi.md:778]:
```suggestion
```

Comment (yahiheb) [reply docs/FAQ/FAQ-UseWasabi.md:778]: I guess this is out of the scope of this PR.

**Review by yahiheb — APPROVED** [review]
ACK. Good job @peterchaudhary 

IMO we should remove this whole question as it does not make much sense now given that auto coinjoin is on by default.

**Comment by peterchaudhary** [comment]
@yahiheb we could remove it, but alternatively, since it is an FAQ, we could just change the answer to something along the lines of: "It's best to coinjoin every time you make a transaction. So, Wasabi has made auto-coinjoin the default option...." We can also direct users to how they can opt out of the coinjoin option, and the costs involved (if there are any) when they do coinjoin.

**Comment by yahiheb** [comment]
> we could remove it, but alternatively, since it is an FAQ, we could just change the answer to something along the lines of: "It's best to coinjoin every time you make a transaction. So, Wasabi has made auto-coinjoin the default option...."

ACK

**Pull request merged**